salad.html

```diff
@@ -40,72 +25,130 @@ <h1>🥗HAESNI'S SALAD🥗</h1>
       <cardsection>
         <div class="card">
           <h3>아보카도쉬림프샐러드</h3>
-          <h5>#생아보카도 #버터새우 #케이준버터</h5>
-          <img class="stuff" src="./assets/saladavocado.jpg" />
-          <img class="btn" src="./assets/heartbtn.png" />
+          <div class="hashtag">
+            <h5>#생아보카도 #버터새우 #케이준버터</h5>
+            <button>+</button>
+          </div>
+          <img
+            class="stuff"
+            src="./assets/saladavocado.jpg"
+            alt="아보카도쉬림프샐러드"
+          />
+          <img class="minebtn" src="./assets/heartbtn.png" alt="찜" />
         </div>
 
         <div class="card">
           <h3>치킨텐더샐러드</h3>
-          <h5>#닭안심 #머스타드소스</h5>
-          <img class="stuff" src="./assets/saladchiken.jpg" />
-          <img class="btn" src="./assets/heartbtn.png" />
+          <div class="hashtag">
+            <h5>#닭안심 #머스타드소스</h5>
+            <button>+</button>
+          </div>
+          <img
+            class="stuff"
+            src="./assets/saladchiken.jpg"
+            alt="치킨텐더샐러드"
+          />
+          <img class="minebtn" src="./assets/heartbtn.png" alt="찜" />
         </div>
 
         <div class="card">
           <h3>와사비크래미샐러드</h3>
-          <h5>#와사비크림 #크래미</h5>
-          <img class="stuff" src="./assets/saladcrap.jpg" />
-          <img class="btn" src="./assets/heartbtn.png" />
+          <div class="hashtag">
+            <h5>#와사비크림 #크래미</h5>
```

Comment:
저도 피드백받은 내용인데 #해쉬태그는 ul li태그로 나열하는 방법도 좋은 것 같아요! 
```suggestion
li {
  list-style-position:inside;
  white-space: nowrap;
  overflow: hidden;
  text-overflow: ellipsis;    
}
```

Reply:
고쳤습니당!! 😄